src/EA.Iws.Web/Areas/NotificationMovements/Controllers/CancelMovementController.cs

```diff
@@ -5,13 +5,15 @@
     using System.Linq;
     using System.Threading.Tasks;
     using System.Web.Mvc;
+    using Core.Authorization.Permissions;
     using Core.Movement;
+    using Infrastructure.Authorization;
     using Prsd.Core.Mediator;
     using Requests.Movement;
     using Requests.Movement.Receive;
     using ViewModels.CancelMovement;
 
-    [Authorize]
+    [AuthorizeActivity(ExportMovementPermissions.CanReadExportMovementsExternal)]
```

Comment:
I think this should be the edit permission, rather than the read permission, since you are changing the state of the movement by cancelling it. (This also means the request should have its permission changed too as that is also using the read permission.)